### ISSUE TITLE
reserve-jumbo looks bad on small screens.

### ISSUE THREAD

**Issue by nathansbradshaw** [UnitedCode/noWhey#4]
Title.

**Comment by glitchedmob** [comment]
This is a known issue. Mobile styling has not been worked on at all. It will most likely be the last step in the project.

**Comment by nathansbradshaw** [comment]
So it is fine if I create the Menus page with no mobile styling, because we will add it in later?

**Comment by glitchedmob** [comment]
Yup that will work. Mobile styling will be the last step. I want to get a handful of devices with different screen sizes together and do thorough mobile testing
